src/Components/ArticleList.jsx

```diff
@@ -1,43 +1,59 @@
-import { useEffect, useState } from "react"
-import { Link } from "react-router-dom"
-import { fetchAllReviews } from "../api"
-export default function ArticleList ({isLoading, setIsLoading}){
-  const [reviews, setReviews] = useState([])
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { fetchAllReviews } from "../api";
+import '../styles/ArticleList.css'
+export default function ArticleList({ isLoading, setIsLoading }) {
+  const [reviews, setReviews] = useState([]);
 
-  useEffect(()=>{
-    setIsLoading(true)
-    fetchAllReviews().then((reviews)=>{
-      setReviews(reviews)
-      setIsLoading(false)
-    })
-  },[setIsLoading])
+  //Initial get req vote state
+  
+  useEffect(() => {
+    setIsLoading(true);
+    fetchAllReviews().then((reviews) => {
+      setReviews(reviews);
+      setIsLoading(false);
+    });
+  }, [setIsLoading]);
 
-  const displayReviews = (review)=>{
+  const displayReviews = (review) => {
     return (
-      <article className="review-card" key={review.review_id}>
+      <article
+        id={review.review_id}
+        className="review-card"
+        key={review.review_id}
+      >
         <div className="review-header">
-          <p> Posted by <span>{review.owner}</span></p>
+          <p>
+            Posted by <span>{review.owner}</span>
+          </p>
           <p>{review.created_at}</p>
         </div>
         <div className="review-body">
           <h3>{review.title}</h3>
           <Link to={`/reviews/${review.review_id}`}>
-          <img src={review.review_img_url} alt={review.title} ></img>
+            <img src={review.review_img_url} alt={review.title}></img>
           </Link>
         </div>
         <div className="review-footer">
-          <p>{review.votes} Votes</p>
+          <button
+            className="default-vote"
+            id={review.votes}
+          >
+            {review.votes} Votes
+          </button>
           <p>Comments</p>
```

Comment:
This doesn't actually do or show anything yet. Did you mean to show the comment count here?